package.json

```diff
@@ -71,5 +71,9 @@
     "webpack-dev-server": "^1.9.0",
     "webpack-error-notification": "^0.1.4"
   },
+  "engines": {
+    "node": ">=8.11.2",
+    "npm": "~5.6.0"
```

Comment:
This is not true. I can get it to work with Node 6, for example. I believe there is a confusion between _expectation_ of the engine being Node 8 because it is LTS, and the definition in the `package.json` of the constraints under which this package will not work.

Reply:
I was assuming that `npm5` was incompatible with `node6`, which is not true, and the intention is to enforce a minimum `npm` version, so I'll change things to reflect that.